scripts/check-markdown-files.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+EXCLUDED_DIRECTORIES="./docs|./samples|.github"
+REPO_README="./README.md"
+CONTRIBUTING_README="./CONTRIBUTING.md"
+
+# Count all readme files in repo not in docs, excluding certain directories, the main repo readme, and the contributing readme
+COUNT_OUT_OF_PLACE_FILES=$(find -iname "*md" ! -iwholename $REPO_README ! -iwholename $CONTRIBUTING_README | egrep -v $EXCLUDED_DIRECTORIES | wc -l)
```

Comment:
super nit: COUNT**OF**

```suggestion
COUNT_OF_OUT_OF_PLACE_FILES=$(find -iname "*md" ! -iwholename $REPO_README ! -iwholename $CONTRIBUTING_README | egrep -v $EXCLUDED_DIRECTORIES | wc -l)
```